source/slang/hlsl.meta.slang

```diff
@@ -11760,7 +11983,9 @@ matrix<int, N, M> sign(matrix<T, N, M> x)
     }
 }
 
-// Sine
+/// Sine
```

Comment:
Needs to end with ".".

Reply:
Done, added a period to any description missing it.